doc/porting/board_porting.rst

```diff
@@ -120,3 +120,25 @@ Board
 A board implements an SoC with all its features, together with peripherals
 available on the board that differentiates the board with additional interfaces
 and features not available in the SoC.
+
+Default board configuration
+***************************
+
+When porting a board, you must provide its default configuration. This default
+configuration could be overridden, or otherwise will be the reference when an
+application, test or sample is compiled for this board.
+
+In order to provide coherency accross the various boards and ease the work of
+users providing applications that are not board specific, following guidelines
+should be followed when porting a board:
+
+- Provide pins and drivers configuration that matches the board valuable
+components, such as sensors, button, leds or communication i/f such as USB,
```

Comment:
Make sure the second line of a list item is indented properly
    
    - Provide pin and driver configurations that match the board's
      valuable components such as sensors, buttons, or LEDs, and communication
      interfaces such as USB, Ethernet connector, or Bluetooth/WiFi chip.